### ISSUE TITLE
Prevent large changesets in Osmose

### ISSUE THREAD

**Issue by SomeoneElseOSM** [osm-fr/osmose-frontend#362]
Would it be possible for the Osmose editor to prevent changesets like https://www.openstreetmap.org/changeset/117752030 and https://www.openstreetmap.org/changeset/118386162 from being created?  These both have:

- very large bbox, lots of objects and clearly no on-the-ground checking by the mapper has taken place,
- no meaningful changeset comment, just "Fixed with Osmose"

I'm aware that https://wiki.openstreetmap.org/wiki/Osmose and the "Welcome to Osmose-QA" text in the editor should make it really clear that it shouldn't be used as the only source of "correctness", but mappers are ignoring those warnings.

**Comment by frodrigo** [comment]
Similar to #164

**Comment by SomeoneElseOSM** [comment]
Yes - it is the same problem. If Osmose can't "police" its users effectively then we (the DWG) will have to be more proactive about warning and blocking Osmose users and reverting their changes where it is obvious that they have no local knowledge.
